src/Microsoft.Health.SqlServer/Features/Health/SqlServerHealthCheck.cs

```diff
@@ -31,22 +31,14 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
     {
         _logger.LogInformation($"Starting {nameof(SqlServerHealthCheck)}.");
 
-        try
-        {
-            using SqlConnectionWrapper sqlConnectionWrapper = await _sqlConnectionWrapperFactory.ObtainSqlConnectionWrapperAsync(cancellationToken);
-            using SqlCommandWrapper sqlCommandWrapper = sqlConnectionWrapper.CreateRetrySqlCommand();
-
-            sqlCommandWrapper.CommandText = "select @@DBTS";
-
-            await sqlCommandWrapper.ExecuteScalarAsync(cancellationToken).ConfigureAwait(false);
-
-            _logger.LogInformation("Successfully connected to SQL database.");
-            return HealthCheckResult.Healthy("Successfully connected.");
-        }
-        catch (Exception ex)
-        {
-            _logger.LogError(ex, "Failed to connect to the data store.");
-            return HealthCheckResult.Unhealthy("Failed to connect.");
```

Comment:
We don't need to catch this. An uncaught exception will return 503 with a generic "Unhealthy" status message